pkg/apis/kops/util/labels.go

```diff
@@ -22,18 +22,21 @@ import (
 	"k8s.io/api/core/v1"
 )
 
-func GetNodeRole(node *v1.Node) string {
-	role := ""
+func GetNodeRoles(node *v1.Node) string {
+	keyRoles := make([]string, 0)
```

Comment:
FYI, it's usually easier to just do `var keyRoles []string` - and you can append to a nil slice (in contrast to a nil map)